introduction.md

```diff
@@ -1,4 +1 @@
 Hello! I am Ryan and I'm here to learn about GitHub. My favorite Code School path is HTML & CSS.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?